style/theme.css

```diff
@@ -0,0 +1,38 @@
+@media screen and (max-width: 767px) {

```

Comment:
theme.css should contain small changes on top of all other normal styling used for temporary purposes (i.e., a holiday theme)

Reply:
I moved the contents from theme.css to modules.css